tests/jballerina-integration-test/src/test/resources/resiliency/src/resiliencyservices/15_http2_failover.bal

```diff
@@ -56,7 +56,7 @@ service delay06 on backendEP06 {
     }
     resource function delayResource(http:Caller caller, http:Request req) {
         // Delay the response for 5000 milliseconds to mimic network level delays.
-        runtime:sleep(5000);
+        runtime:sleep(10000);
```

Comment:
Just for clarity how does increasing the delay help?
We need to update the comment as well.